exhibitor-core/src/main/java/com/netflix/exhibitor/core/azure/AzureCredential.java

```diff
@@ -0,0 +1,6 @@
+package com.netflix.exhibitor.core.azure;
+
+public interface AzureCredential {
```

Comment:
Credentials could come in through one of several different paths. The S3 implementation includes this, but I didn't. Leaving as-is for consistency and so we don't have to change as much :)